exercises/accumulate/README.md

```diff
@@ -21,9 +21,8 @@ Check out the test suite to see the expected function signature.
 
 ## Restrictions
 
-Keep your hands off for comprehension functionality and any methods that accept
-a function as a parameter (`map`, `flatMap`, `fold`, etc.)
-provided by the standard library!
+Keep your hands off that collect/map/fmap/whatchamacallit functionality
```

Comment:
@ppartarr Maybe you could revert this bit to the old version, as that is a bit more Scala-specific?

Reply:
This is fixed in 22863b1 :+1: